recipe/meta.yaml

```diff
@@ -12,14 +12,14 @@ package:
 
 source:
   # latest commit from branch {{ cctools_version }}-ld64-{{ ld64_version }}
-  - url: https://github.com/tpoechtrager/cctools-port/archive/dbcdc0d36eeb47a7ac4b7d5abe772886fec52e56.tar.gz
-    sha256: 24501b49a0cc0ec13451a36a62e6016a83a1fc8b3fdc818c113ac2a6b6d486f6
+  - url: https://github.com/tpoechtrager/cctools-port/archive/236a426c1205a3bfcf0dbb2e2faf2296f0a100e5.tar.gz
+    sha256: f7c73122ebb0ea2cf094639b9ae387857d528337fea3d07d9800d9d0980241af
     patches:
       - patches/dont_link_with_libcxxabi.patch
       - patches/530-ld64-add-conda-specific-env-vars-to-modify-lib-search-paths.patch
 
 build:
-  number: 22
+  number: 0
```

Comment:
not sure if resetting the build number is correct for this feedstock, but since all relevant versions are being bumped, I assume so...